wiki/en/en-Server-Win-Mac.md

```diff
@@ -24,13 +24,13 @@ permalink: "/wiki/Server-Win-Mac"
 
 1. Type in the name, city and country so that other users can easily identify you.
 
-1. Specify a musical genre if you wish. This will make your server appear in the relevant server list for clients (see more [details on Central Servers](Central-Servers)).
+1. Specify a music genre if you wish. This will make your server appear in the relevant server list for clients (see more [details on Central Servers](Central-Servers)).
```

Comment:
I'd remove "if you wish".  "All Genres 1" is full and anyone should be looking at the server lists (using Jamulus Explorer) to find space and an appropriate genre.

Reply:
Done